src/modules/apps/install.ts

```diff
@@ -24,11 +25,32 @@ const promptPolicies = async () => {
   return promptConfirm('Do you accept all the Terms?')
 }
 
+const hasLicenseFile = async (name: string, version: string) => {
+  const client = createRegistryClient()
+  try {
+    await client.getAppFile(name, version, '/public/metadata/licenses/en-US.md')
+    return true
+  } catch (err) {
+    if (err.response?.status === 404) {
+      return false
+    }
+    throw err
+  }
+}
+
+const licenseURL = async (app: string, termsURL?: string): Promise<string | undefined> => {
+  const [name, argVersion] = app.split('@')
+  const version = argVersion ?? 'x'
+  return (await hasLicenseFile(name, version)) ? `https://extensions.myvtex.com/_v/terms/${name}@${version}` : termsURL
```

Comment:
```suggestion
return (await hasLicenseFile(name, version)) ? `https://apps.vtex.com/_v/terms/${name}@${version}` : termsURL
```

Reply:
Done in 6dcee75b3c884f4f750ee7ca20680fff37b73dff, thanks!